src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -148,6 +164,8 @@
     {{ settlementAgreementRejectedForClaimantDashboard(claim) }}
   {% elif status === ClaimStatus.PART_ADMIT_PAY_IMMEDIATELY %}
     {{ partAdmittedClaimantAcceptsClaimantDashboard(claim) }}
+  {% elif status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ %}
+    {{ claimRejectedFreeMediationNOWithoutDQsForClaimantDashboard() }}
```

Comment:
I don't see CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE_NO_DQ this scenario, please check if it's required.

Reply:
this is redundant. it would just be claim settled and would be covered under CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE. I shall remove CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE_NO_DQ

Reply:
ok